docs/api.md

```diff
@@ -156,21 +127,65 @@ bpolys = {
         ]
     ],
 }
+layer = {
+    "name": "My layer name"
+    "description": "My layer description"
+    "data": {
+        "result": [
+            {"timestamp": "2014-01-01T00:00:00Z", "value": 4708},
+            {"timestamp": "2014-02-01T00:00:00Z", "value": 4842},
+            {"timestamp": "2014-03-01T00:00:00Z", "value": 4840},
+            {"timestamp": "2014-04-01T00:00:00Z", "value": 4941},
+            {"timestamp": "2014-05-01T00:00:00Z", "value": 4987},
+            {"timestamp": "2014-06-01T00:00:00Z", "value": 5007},
+            {"timestamp": "2014-07-01T00:00:00Z", "value": 5020},
+            {"timestamp": "2014-08-01T00:00:00Z", "value": 5168},
+            {"timestamp": "2014-09-01T00:00:00Z", "value": 5355},
+            {"timestamp": "2014-10-01T00:00:00Z", "value": 5394},
+            {"timestamp": "2014-11-01T00:00:00Z", "value": 5449},
+            {"timestamp": "2014-12-01T00:00:00Z", "value": 5470},
+            {"timestamp": "2015-01-01T00:00:00Z", "value": 5475},
+            {"timestamp": "2015-02-01T00:00:00Z", "value": 5477},
+            {"timestamp": "2015-03-01T00:00:00Z", "value": 5481},
+            {"timestamp": "2015-04-01T00:00:00Z", "value": 5495},
+            {"timestamp": "2015-05-01T00:00:00Z", "value": 5516},
+            {"timestamp": "2015-06-01T00:00:00Z", "value": 5517},
+            {"timestamp": "2015-07-01T00:00:00Z", "value": 5519},
+            {"timestamp": "2015-08-01T00:00:00Z", "value": 5525},
+            {"timestamp": "2015-09-01T00:00:00Z", "value": 5560},
+            {"timestamp": "2015-10-01T00:00:00Z", "value": 5564},
+            {"timestamp": "2015-11-01T00:00:00Z", "value": 5568},
+            {"timestamp": "2015-12-01T00:00:00Z", "value": 5627},
+            {"timestamp": "2016-01-01T00:00:00Z", "value": 5643},
+            {"timestamp": "2016-02-01T00:00:00Z", "value": 5680},
+            {"timestamp": "2016-03-01T00:00:00Z", "value": 5681},
+            {"timestamp": "2016-04-01T00:00:00Z", "value": 5828},
+            {"timestamp": "2016-05-01T00:00:00Z", "value": 5974},
+            {"timestamp": "2016-06-01T00:00:00Z", "value": 5990},
+            {"timestamp": "2016-07-01T00:00:00Z", "value": 5991},
+            {"timestamp": "2016-08-01T00:00:00Z", "value": 5997},
+            {"timestamp": "2016-09-01T00:00:00Z", "value": 6002},
+            {"timestamp": "2016-10-01T00:00:00Z", "value": 6010},
+            {"timestamp": "2016-11-01T00:00:00Z", "value": 6010},
+            {"timestamp": "2016-12-01T00:00:00Z", "value": 6016},
+            {"timestamp": "2017-01-01T00:00:00Z", "value": 6015},
+        ]
+    }
+}
 parameters = {
     "name": "GhsPopComparisonBuildings",
-    "layerName": "building_count",
     "bpolys": bpolys,
+    "layer": layer,
+    "includeSvg": False,  # Optional
+    "includeHtml": False,  # Optional
 }
 response = requests.post(url, json=parameters)
 assert response.status_code == 200
```

Comment:
see above

Reply:
Removed the assert statement as it is not necessary.